CMakeLists.txt

```diff
@@ -222,6 +222,7 @@ if(VLLM_GPU_LANG STREQUAL "CUDA")
     "csrc/quantization/gptq_marlin/awq_marlin_repack.cu"
     "csrc/quantization/gguf/gguf_kernel.cu"
     "csrc/quantization/fp8/fp8_marlin.cu"
+    "csrc/quantization/quant_llm/quant_llm_linear.cu"
```

Comment:
quant_llm is a general name that doesn't seem related much to fpN quantization, would you be open to renaming? I kind of like the `fp_eXmY` or `fpN_weights` names in other places

Reply:
I'm open to renaming it to something else. Maybe keep `fp_eXmY` for consistency?